apache2/msc_logging.c

```diff
@@ -1174,6 +1174,9 @@ void sec_audit_logger_json(modsec_rec *msr) {
             yajl_kv_bool(g, "response_body_dechunked", 1);
         }
 
+#ifdef LOG_NO_PRODUCER
+        if (msr->txcfg->debuglog_level < 9) return;
```

Comment:
Indeed, seems to be part of another patch.
Do I recommit? It seems I cannot amend (actually, I can but it detects no difference if I try)